### DISCUSSION TITLE
XmlException

### DISCUSSION THREAD

**Discussion by mspace23** [akunzai/GSS.Authentication.CAS#221] [Q&A]
Hello

I read that I have to ensure that https should be configure so the log in from CAS to be enable.
Now I have another issue...

I get

An unhandled exception occurred while processing the request.
XmlException: '=' is an unexpected token. The expected token is ';'. Line 32, position 91.

**Comment by akunzai** [comment]
That means your CAS server not response with correnct XML format.

**Comment by mspace23** [comment]
I am running my app in localhost. Is something that I have to set up with that?
thank you

**Reply by akunzai** [reply]
1. What CAS protocol version are you using and is that supported by your CAS server?
2. It will be helpful if you can tell us the CAS server validation response text  by debug or capture it by using network sniffers(like [Wireshark](https://www.wireshark.org/)).
